packages/babel-parser/test/fixtures/typescript/static-blocks/invalid-static-block-with-accessibility-private/input.ts

```diff
@@ -0,0 +1,3 @@
+class Foo {
+  static private {}
```

Comment:
We can report a better error, like `Static class blocks cannot have any modifier`